gcloud-java-examples/src/main/java/com/google/gcloud/examples/Stat.java

```diff
@@ -0,0 +1,104 @@
+package com.google.gcloud.examples;
```

Comment:
I think between the time that this branch was created and now, we've changed the packaging of examples to include the service name (e.g. we've moved datastore examples from com.google.gcloud.examples to com.google.gcloud.examples.datastore).  Do you mind adding another folder "nio" and changing this package name to `com.google.gcloud.examples.nio` or something like that?